packages/animations/example/lib/main.dart

```diff
@@ -1,3 +1,88 @@
-import 'package:flutter/widgets.dart';
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
 
-void main() => runApp(const Placeholder());
+import 'package:flutter/material.dart';
+
+import 'shared_axis_transition.dart';
+
+void main() {
+  runApp(
+    MaterialApp(
+      theme: ThemeData.from(
+        colorScheme: const ColorScheme.light(),
+      ).copyWith(
+        pageTransitionsTheme: const PageTransitionsTheme(
+          builders: <TargetPlatform, PageTransitionsBuilder>{
+            TargetPlatform.android: ZoomPageTransitionsBuilder(),
+          },
+        ),
+      ),
+      home: _TransitionsHomePage(),
+    ),
+  );
+}
+
+class _TransitionsHomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Material Transitions')),
+      body: Column(
+        children: <Widget>[
+          _TransitionListTile(
+            title: 'Shared Axis',
+            subtitle: 'Page transition where outgoing and incoming '
+                'elements share a fade transition',
+            onTap: () {
+              Navigator.of(context).push(
+                MaterialPageRoute<void>(
+                  builder: (BuildContext context) => SharedAxisTransitionDemo(),
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _TransitionListTile extends StatelessWidget {
+  const _TransitionListTile({
+    this.onTap,
+    this.title,
+    this.subtitle,
+  });
+
+  final Function onTap;
```

Comment:
Type this stronger as GestureTapCallback? (the actual type wanted by ListTile.onTap)